### PR TITLE
unique -> uniqueWithRetry Fixes #27

### LINKED ISSUE #27: cy.get use id if available
IMHO it would make tests less brittle if cy.get would use element ids where available.

accesses such as
```js
    cy.get(':nth-child(1) > :nth-child(1) > :nth-child(1) > div > input').type('foo@bar.com');
```  

might fail easily when layout changes.

```js
    cy.get('#emailInput').type('foo@bar.com');

```

might be better

### PR DESCRIPTION
**Pull request by burner**
uniqueWithRetry tries to get an HTML element by ID to make tests less brittle.
unique fails if IDs are invalid HTML, which can still be displayed by browsers.

also makes typescript and the linter happier.

### CODE REVIEW

File: src/utils/Dictionary.ts [hunk 1]
```diff
@@ -132,13 +132,12 @@ export function getCodeFromEvent(event: AllFdEvents, options?: Options): string
 
 export function getCode(suite: string, description: string, events: AllFdEvents[], options?: Options) {
     let firstVisitUrl = '';
-    for (let i = 0; i < events.length; ++i) {
-        const event = events[i];
+    for (const event of events) {
```
Comment (willemliu) [inline src/utils/Dictionary.ts:135]: Wow! I can't believe I wrote that piece of code. Thanks for doing the code-review as well :relieved:

**Review by willemliu — APPROVED** [review]
Looking good!

**Comment by burner** [comment]
@willemliu thank you, could you also create a new release for the chrome plugin store thing, so I can kick my custom version to the curb?

**Comment by willemliu** [comment]
@burner absolutely. I'm going to do some smoke-testing and after that I'll bundle a new release for Chrome.

**Comment by burner** [comment]
@willemliu perfect thank you

**Pull request merged**